community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/ast/convert/commands/ExpressionConverters.scala

```diff
@@ -47,6 +47,8 @@ object ExpressionConverters {
       case e: ast.NotEquals => e.asCommandNotEquals
       case e: ast.RegexMatch => e.asCommandRegex
       case e: ast.In => e.asCommandIn
+      case e: ast.Like => e.asCommandRegex
+      case e: ast.NotLike => e.asCommandNegatedRegex
```

Comment:
I think we should not compile `NotLike` to commands but rewrite it to `Not(Like)` as for `NotEquals` -> `Not(Equals)`.

The compilation of `NotEquals` to commands is still there because of legacy compatibility, Ronja does not uses it.